README.md

```diff
@@ -17,6 +17,8 @@ Feel free to **Star** and **Watch** it, but watch out for falling debris.
 
 Feedback and contributions are always welcome and appreciated.
 
+[![Gitter](https://img.shields.io/gitter/room/DAVFoundation/DAV-Contributors.svg?style=flat-square)](https://gitter.im/DAVFoundation/DAV-Contributors)
```

Comment:
The button should be right under the first headline of the page. You can see this on https://raw.githubusercontent.com/DAVFoundation/missioncontrol/master/README.md

I didn't notice this in my previous review.